src/sys/unix/mod.rs

```diff
@@ -1,3 +1,17 @@
+/// Helper macro to execute a system call that returns an `io::Result`.
+//
+// Macro must be defined before any modules that uses them.
+macro_rules! syscall {
+    ($e:expr) => {{
+        let res = unsafe { $e };
```

Comment:
Ok, I see what you mean... I want to backtrack on my earlier thoughts :)

If the `unsafe` block is contained in the macro, I **do** think the macro should inject `libc::` in order to prevent passing arbitrary code blocks into the `unsafe` block.

Or, I would vote to remove `unsafe` from the macro.

Reply:
I added the unsafe block in the macro to attempt to make some of the longer calls (i.e. `kevent`) fit on a single line. I'm fine either either option.

@asomers any thoughts?

Reply:
My vote is to move `libc::` into the macro and keep `unsafe` in the macro (your original suggestion).

Reply:
Changed it.